src/devices/tuya.ts

```diff
@@ -540,6 +540,7 @@ const definitions: Definition[] = [
             {vendor: 'TuYa', model: 'iH-F001'},
             {vendor: 'Tesla Smart', model: 'TSL-SEN-DOOR'},
             {vendor: 'Cleverio', model: 'SS100'},
+            tuya.whitelabel('Niceboy', 'ORBIS Windows & Door Sensor', 'Door sensor', ['_TZ3000_qrldbmfn']),
```

Comment:
Instead of `ORBIS Windows & Door Senso` use the model number which is written on the device.

Reply:
It is actually what is listed in both the manual and on the device itself as "model". I did not find any other number that I could use. It is the same for all of the sensors.
![20230726_190921](https://github.com/Koenkk/zigbee-herdsman-converters/assets/5121526/66baf87e-cffa-4c67-9632-a9c5d28ddb61)